src/parser.rs

```diff
@@ -185,9 +190,6 @@ fn parse_hunk_start(line: &str) -> Result<Option<(u64, u64)>> {
         // For the side that is zero, we allow the "UB" of underflowing when the
         // caller subtracts 1 from the result. That is OK b/c we will never use that
         // value, as you cannot comment on text that does not exist.
-        if hunk_start_line_left + hunk_start_line_right == 0 {
-            bail!("Both hunks lines start at 0");
-        }
```

Comment:
Can you also please update the comment above this? Maybe just delete the parts that mention at least one side has to be !0.

Reply:
I left this as a reminder to myself to double check. Will adjust the comment accordingly once verified there are no negative side effects. ETA tomorrow